src/test/java/com/github/junit5docker/DefaultDockerClientIT.java

```diff
@@ -90,7 +89,8 @@ public void ensureImageIsPulled() {
             @Test
             @DisplayName("start a container without ports")
             public void shouldStartContainer() {
-                String containerId = defaultDockerClient.startContainer(WANTED_IMAGE, emptyMap());
+                String containerId = defaultDockerClient.startContainer(WANTED_IMAGE, emptyMap(), null)
```

Comment:
Could we avoid using `null` ? An empty array would be fine for me (declared in constants ?)

Reply:
👍